qcodes/monitor/monitor.py

```diff
@@ -267,7 +266,7 @@ def show() -> None:
     os.chdir(STATIC_DIR)
     try:
         log.info("Starting HTTP Server at http://localhost:%i", SERVER_PORT)
-        with socketserver.TCPServer(("", SERVER_PORT),
```

Comment:
I don't think this change is needed. There is a bug in typing shed with mypy 0.761 that breaks the types around here. I have submitted a pr for that here so it should be fixed in the next version. https://github.com/python/typeshed/pull/3579/files this is why qcodes is checked with mypy 0.750 btw

Reply:
Haha sounds good 😄 do you think it's better to revert this change? I am happy for you to do that, or I can too :P

Reply:
Yes if you can confirm that you only made the change to keep mypy happy lets revert it. I am happy to do so

Reply:
Awesome, I just left work so feel free :+1: